### DISCUSSION TITLE
Blocked by Norton due to the FileRepPup threat

### DISCUSSION THREAD

**Discussion by stlecho** [cargo-lambda/cargo-lambda#712] [Q&A]
When I execute on Windows the "cargo lambda deploy" command, it gets blocked by Norton (see attached screenshot) due to the FileRepPup threat. Is it ok to mark it as "false positive" ?

![image](https://github.com/user-attachments/assets/22d2de85-3dcc-4094-9168-97a0d6f201c9)

**Comment by calavera** [comment]
Yes, this is a false positive.
